sdk/maps/azure-maps-search/src/test/java/com/azure/maps/search/TestUtils.java

```diff
@@ -0,0 +1,307 @@
+import com.azure.maps.search.implementation.models.ReverseSearchCrossStreetAddressResultPrivate;
+import com.azure.maps.search.implementation.models.SearchAddressResultPrivate;
+import com.azure.maps.search.models.BatchReverseSearchResult;
+import com.azure.maps.search.models.BatchSearchResult;
+import com.azure.maps.search.models.PointOfInterestCategoryTreeResult;
+import com.azure.maps.search.models.Polygon;
+import com.azure.maps.search.models.ReverseSearchAddressResult;
+import com.azure.maps.search.models.ReverseSearchCrossStreetAddressResult;
+import com.azure.maps.search.models.SearchAddressResult;
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.provider.Arguments;
+
+
+public class TestUtils {
+
+    static final String FAKE_API_KEY = "1234567890";
+    public static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(30);
+
+    static Polygon getPolygon(File file) {
+        SerializerAdapter jacksonAdapter = JacksonAdapter.createDefaultSerializerAdapter();
+        TypeReference<PolygonPrivate> interimType = new TypeReference<PolygonPrivate>(){};
+        byte[] data = null;
+        try {
+            data = Files.readAllBytes(file.toPath());
+        } catch (IOException e) {
+            Assertions.fail("Unable to find polygon json file");
+        }
+        PolygonPrivate polygonPrivate = null;
+        try {
+            polygonPrivate = jacksonAdapter.<PolygonPrivate>deserialize(data, interimType.getJavaType(),
+               SerializerEncoding.JSON);
+        } catch (IOException e) {
+            Assertions.fail("Unable to parse polygon json file");
+        }
+        Polygon polygon = Utility.toPolygon(polygonPrivate);
+        return polygon;
+    }
+
+    static List<Polygon> getMultiPolygonsResults() {
+        List<Polygon> result = new ArrayList<>();
+        File file1 = new File("src/main/resources/polygon1.json");
+        Polygon polygon1 = TestUtils.getPolygon(file1);
+        File file2 = new File("src/main/resources/polygon2.json");
+        Polygon polygon2 = TestUtils.getPolygon(file2);
+        result.add(polygon1);
+        result.add(polygon2);
+        return result;
+    }
+
+    static SearchAddressResult getSearchAddressResult(byte[] data) {
+        SerializerAdapter jacksonAdapter = JacksonAdapter.createDefaultSerializerAdapter();
+        TypeReference<SearchAddressResultPrivate> interimType = new TypeReference<SearchAddressResultPrivate>(){};
+        SearchAddressResultPrivate searchAddressResultPrivate = null;
+        try {
+            searchAddressResultPrivate = jacksonAdapter.<SearchAddressResultPrivate>deserialize(data, interimType.getJavaType(),
+               SerializerEncoding.JSON);
+        } catch (IOException e) {
+            Assertions.fail("unable to parse file");
```

Comment:
This has the same effect of simply throwing the exception, which is not what you should do. Just remove all try/catch in this file and let the test itself (which is calling these methods) deal with the exception!